blog-page.html

```diff
@@ -0,0 +1,285 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="/src/output.css" rel="stylesheet">
+  <link href="/src/input.css" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+  <title>TravelUp - About</title>
+  <style>
+    /* Ensure no margin/padding on body */
+    body {
+      margin: 0;
+      padding: 0;
+    }
+
+    /* Style for hero section */
+    header {
+      position: relative;
+      width: 100%; /* Ensure the hero section spans the whole width */
+    }
+
+    /* Style for hero image */
+    header img.hero-image {
+      width: 100%;
+      height: auto; /* Maintain aspect ratio */
+      display: block; /* Ensure the image behaves as a block element */
+    }
+
+    /* Style for hero text */
+    .hero-text {
+      position: absolute;
+      top: 50%; /* Position at the vertical center */
+      left: 50%; /* Position at the horizontal center */
+      transform: translate(-50%, -50%); /* Center the text */
+      color: white;
+      font-size: 3em;
+      text-align: center;
+      background-color: rgba(0, 0, 0, 0.5); /* Add a semi-transparent background */
+      padding: 20px;
+      border-radius: 10px;
+    }
+
```

Comment:
Wont need this once we have used the original index.html header

Don't need any overlay text on the hero image